modules/ROOT/pages/migrate-ah.adoc

```diff
@@ -0,0 +1,44 @@
+:experimental:
+
+= Migrating from Atomic Host (AH) to Fedora CoreOS (FCOS)
+
+== Overview
+
+https://www.projectatomic.io/[Atomic Host] is a system to deploy applications in containers. Atomic Hosts users are encouraged to migrate to FCOS, as the project is being folded into FCOS.
```

Comment:
- Can we use Fedora Atomic Host to start the sentence?
- should we switch to past tense? replacing the first `is` with `was`
- Maybe start the 2nd sentence with `Existing Fedora Atomic Host users`